### ISSUE TITLE
How to add alldebrid token?

### ISSUE THREAD

**Issue by sharpordie** [Goldenfreddy0703/Otaku#381]
Is there a way to set alldebrid token directly in a settings file or db? I created a small shell script to setup my kodi and I would like to automate this as well.

Thank you guy for the amazing work.

**Comment by joezito129** [comment]
you can find a settings.xml file in the ../userdata/{addon.id}
userdata directory is based on your operating system https://kodi.wiki/view/Userdata
add your token to setting id='alldebrid.apikey'

**Comment by joezito129** [comment]
i dont think alldebrid works as they removed the api call to check cached stuff only cloud would work

**Comment by Gujal00** [comment]
@joezito129  alldebrid works fine, we cant check for cached torrent, but is now coded to check each source while playing and skips until it finds a cached one and plays

**Comment by sharpordie** [comment]
> you can find a settings.xml file in the ../userdata/{addon.id} userdata directory is based on your operating system https://kodi.wiki/view/Userdata add your token to setting id='alldebrid.apikey'

I don't find the .apikey setting, should I add the whole setting node?
https://github.com/Goldenfreddy0703/Otaku/blob/main/plugin.video.otaku/resources/settings.xml#L332

**Comment by joezito129** [comment]
Yes it should be added by the Otaku Team and then we can push a release with that setting id. Also the setting.xml that you should modify is in ../userdata/addon_data/plugin.video.otaku

**Comment by sharpordie** [comment]
I don't think it's a good idea to put the apitoken in plain text either.

I could invoke RunPlugin(plugin://plugin.video.otaku/authAllDebrid) from JSON-RPC, copy the pairing code from logs and curl the alldebrid api with my credentials in my shell script.

**Comment by joezito129** [comment]
you can set the alldebrid.apikey in Otaku by running this command:

xbmcaddon.Addon('plugin.video.otaku').setSetting('alldebrid.apikey', {token})

so you dont have to modify any xml files

**Comment by sharpordie** [comment]
OK, thank you, but I am not using Python, I am using a Bash script from SSH.
Is it doable using JSON-RPC with curl?
